MyProjects/GeneratePasswords/GeneratePasswords/App.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace GeneratePasswords
+{
+    internal class App
+    { 
+        
+        private const string Lowercase = "abcdefghijklmnopqrstuvwxyz";
+        private const string Uppercase = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        private const string listNumbers = "0123456789";
+        private const string specialChars = "!@#$%^&*()_+-=[]{}|;:'\"<>,.?/";
```

Comment:
You can use REgex() for this purpose. https://learn.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.match?view=net-7.0